src/components/StaffPayment/StaffPayment.vue

```diff
@@ -230,10 +233,10 @@ export default class StaffPayment extends Vue {
   @Emit('update:staffPaymentData')
   private emitStaffPaymentData ({
     option = this.staffPaymentData.option,
-    routingSlipNumber = this.staffPaymentData.routingSlipNumber || '',
-    bcolAccountNumber = this.staffPaymentData.bcolAccountNumber || '',
-    datNumber = this.staffPaymentData.datNumber || '',
-    folioNumber = this.staffPaymentData.folioNumber || '',
+    routingSlipNumber = this.staffPaymentData.routingSlipNumber?.trim() || '',
```

Comment:
Does it need to be trimmed here as well if it is handled on the input?
I don't see a problem with this, just curious! 

Does Folio number need trim on the model in the template as well?

Reply:
I have them both in because the vmodel.trim trims the leading spaces in real time (this avoids validation messages like must have 9 characters or whatnot).
What vmodel.trim doesn't seem to do is trim trailing spaces. Even if you tab/click out of the field, which is why I left the manual trim in as well.

Reply:
Sounds good!